nixos/modules/services/misc/gitlab.nix

```diff
@@ -475,6 +526,7 @@ in {
         gitAndTools.git
         openssh
         nodejs
+        procps
```

Comment:
That is addressed in the `postPatch` phase in the Gitlab package, maybe there is where an update is needed?